packages/snaps-jest/src/types.ts

```diff
@@ -297,5 +297,3 @@ export type Snap = {
 };
 
 export type SnapResponse = Infer<typeof SnapResponseStruct>;
-
-export { NotificationType } from '@metamask/snaps-rpc-methods';
```

Comment:
Do we not want to re-export this anymore?

Reply:
Probably not, if we assume everyone will use `snaps-sdk`?